packages/publish-views/pipeline/to-file.ttl

```diff
@@ -7,15 +7,15 @@
     [
       :stepList
         (
-          <ToNtriples> _:save
+          <#loadViews> <#prepareView> <#pointerToDataset> <#validate> <#checkShaclReports> <#flatten> <#serialize> _:save
```

Comment:
Inlined the steps instead of pipeline because they were not propagating the error somehow and would cause mocha process to hang

Reply:
I'm trying to remember once I had a problem reusing one pipeline in many pipelines, ... difficult to recall :/